src/dom.ts

```diff
@@ -271,3 +272,54 @@ export function compareElementArea(a: HTMLElement, b: HTMLElement): number {
 
     return aArea - bArea
 }
+
+/** DOM reference to the last used Input field
+ */
+//#content_helper
+let LAST_USED_INPUT: HTMLElement = null
+
+export function getLastUsedInput () {
+   return LAST_USED_INPUT
+}
+
+/** WARNING: This function can potentially recieve malicious input! For the
+ *  whole discussion about this, see
+ *  https://github.com/cmcaine/tridactyl/pull/225
+ *
+ *  Remember to check whether WebComponents change anything security-wise:
+ *  https://developer.mozilla.org/en-US/docs/Web/Web_Components/Custom_Elements
+ *  https://bugzilla.mozilla.org/show_bug.cgi?id=1406825
+ * */
+function onPageFocus(elem: HTMLElement, args: any[]): void {
+    if (isTextEditable(elem)) {
+        LAST_USED_INPUT = elem
+        config.getAsync("allowautofocus").then((allow) => {
+            if (allow)
```

Comment:
Everywhere this is used, it should be changed to a string comparison: `if (allow == "true")`